Source/AdColonyAdapterConfiguration.swift

```diff
@@ -4,18 +4,23 @@
 // license that can be found in the LICENSE file.
 
 import AdColony
+import os.log
 
 /// A list of externally configurable properties pertaining to the partner SDK that can be retrieved and set by publishers.
 @objc public class AdColonyAdapterConfiguration: NSObject {
 
+    private static var log = OSLog(subsystem: "com.chartboost.mediation.adapter.adcolony", category: "Configuration")
+
     /// Flag that can optionally be set to enable the partner's test mode.
     /// Disabled by default.
     @objc public static var testMode: Bool = false {
         didSet {
             let options = AdColonyAdapter.options
             options.testMode = testMode
             AdColony.setAppOptions(options)
-            print("AdColony SDK test mode set to \(testMode)")
+            if #available(iOS 12.0, *) {
+                os_log(.debug, log: log, "AdColony SDK test mode set to %{public}s", testMode ? "true" : "false")
```

Comment:
nit: not sure how this works, but would it be straightforward to use the Bool default description like we were doing in the print?
E.g. something like `os_log(.debug, log: log, "AdColony SDK test mode set to %{public}s", testMode)`
maybe changing the `%{}s` to something else idk